scripts/library.sh

```diff
@@ -502,6 +502,22 @@ function get_latest_knative_yaml_source() {
   fi
 }
 
+# Returns the latest gke version in [major].[minor].[patch] with provided prefix
```

Comment:
Move to e2e-tests.sh, since this is pretty specific.

Reply:
done